lib/Catalyst/Controller/Swagger.pm

```diff
@@ -46,8 +46,8 @@ sub api_docs : Local {
            my @path;
            for my $chain_part (@{$expanded->chain}) {
              push @path, join('/',$chain_part->attributes->{PathPart}[0],
-                              ($chain_part->attributes->{CaptureArgs} ? join('/',('*') x $chain_part->attributes->{CaptureArgs}[0]) : ()),
-                              ($chain_part->attributes->{Args} ? join('/',('*') x $chain_part->attributes->{Args}[0]) : ()),
+                              ($chain_part->attributes->{CaptureArgs}[0] ? join('/',('*') x $chain_part->attributes->{CaptureArgs}[0]) : ()),
```

Comment:
I just made it work without warning. You're right: it's not the best solution ...